backend/tests/routers/gameRouter.test.ts

```diff
@@ -0,0 +1,33 @@
+import supertest from 'supertest';
+import app from '../../src/app';
+
+const request = supertest(app);
+
+const game = {
+	name: 'Test Game',
+	description: 'Test Description',
+	platform: 'Test Platform',
+	releaseDate: '2023-04-13',
+	playtime: '60'
+};
+
+describe('Testing gameRouter.ts', () => {
+	test('Test adding a platform', async () => {
```

Comment:
We need to figure out a way to discard the test data after the test is done. If we don't manually remove the "Test Platform" from the db in between runs it will fail to re-add it.

Reply:
Methods to delete games from the db feels like it should be its own user story, I'll add it to the backlog